features/eolearn/features/feature_manipulation.py

```diff
@@ -34,51 +37,55 @@ class SimpleFilterTask(EOTask):
     A filter_func is a callable which takes a numpy array and returns a bool.
     """
 
-    def __init__(self, feature, filter_func, filter_features=...):
+    def __init__(self, feature, filter_func: Callable[[_T], bool], filter_features=...):
         """
         :param feature: Feature in the EOPatch , e.g. feature=(FeatureType.DATA, 'bands')
-        :type feature: (FeatureType, str)
+        :type feature: (FeatureType, str) or FeatureType.TIMESTAMP
```

Comment:
why not add this to signature as well? then please also remove types from docstrings so we dont confuse docbuilders

Reply:
i know that the last argument cannot be typed properly yet, just slap `Any` on it for now i guess...

Reply:
Added types. Btw, for Sphinx docs types in docstrings take priority over type hints so it would just take those.